controllers/lvmcluster_controller.go

```diff
@@ -79,6 +79,7 @@ type LVMClusterReconciler struct {
 //+kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,verbs=get;create;update;delete
 //+kubebuilder:rbac:groups=topolvm.cybozu.com,resources=logicalvolumes,verbs=get;list;watch
 //+kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;watch
+//+kubebuilder:rbac:groups="",resources=nodes,verbs=get;list;watch
```

Comment:
Is not get and list would be enough?

Reply:
I though it would  but I was getting issues around `watch`. Don't have those logs right now. Adding `watch` as well fixed it.